packages/bpk-docs/src/pages/BadgePage/BadgePage.js

```diff
@@ -91,8 +204,9 @@ const BadgePage = () => (
     blurb={[
       <Paragraph>
         Badges are labels which hold small amounts of information. They are
-        currently available in one colour. Badges are most often used as
-        counters, such as an indication of unread notifications.
+        available in a number of colours to signify different meanings. Badges
```

Comment:
Sadly, `colours` needs to be changed to the Americanised `colors` to match the rest of the docs (and this page).

Reply:
😞